API/Models/AnswerDTO.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
```

Comment:
Why did you create a new class instead of just updating https://github.com/KamandaX/RecommendationAPI/blob/master/API/Models/QuestionOption.cs ?

Reply:
Same as with `GetQuestionDTO`

Reply:
Could you rename either QuestionOption or AnswerDTO class to be consistent?

Reply:
Yes. Renamed.